package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "crowdbrew-frontend",
-  "version": "0.0.0",
+  "version": "0.0.1",
```

Comment:
a minor thing, but according to semver it should be 0.1.0: http://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase